frontend/src/components/document/TableView/TableEntry.jsx

```diff
@@ -103,8 +113,12 @@ export const TableEntry = ({ redaction, updateParentState }) => {
             );
           }}
         />
-        <img src={CommentIcon} />
-        <img src={NavigateToIcon} />
+        <img
+          src={NavigateToIcon}
+          onClick={() => {
+            popOutToDocView();
```

Comment:
do we also want to set selection to the current text node?

Reply:
Lol I was thinking about it but it seems rather contrived because we don't have any reference to the editor in the table view so we can't get our hands on the node, and then the "pop out" callback is passed in from DocumentPage which also doesn't have any reference to the editor so we'd have to lift editor state out of `TextEditor` which sounds like it would break a lot of things